### PR TITLE
Clone gov contracts as much as possible from mainnet

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

**Review by scott-silver — APPROVED** [review]
🥳

File: .gitignore [hunk 1]
```diff
@@ -10,9 +10,7 @@ artifacts/
 build/
 cache/
 dist/
-deployments/*/*/artifacts
-deployments/*/*/cache
-deployments/*/*/contracts
+deployments/.contracts
```
Comment (scott-silver) [inline .gitignore:13]: do we need to update the files deleted in `yarn spider --clean`, too?

Comment (jflatow) [reply .gitignore:13]: Probably, didn't know about that command -- where is it?

File: scenario/context/CometContext.ts [hunk 2]
```diff
@@ -215,73 +206,52 @@ export class CometContext {
...
+        signatures,
+        calldatas,
+        'FastExecuteProposal',
+        { gasPrice: 0 }
+      )
+    ).wait();
+    const proposeEvent = proposeTxn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId, , , , , , startBlock, endBlock] = proposeEvent.args;
+
+    await this.executePendingProposal(proposalId, startBlock, endBlock);
```
Comment (kevincheng96) [inline scenario/context/CometContext.ts:254]: 🧼

**Review by kevincheng96 — APPROVED** [review]
I reviewed the diffs between this PR and #517 and everything LGTM 👍

File: .gitignore (hunk 1 above)
Comment (scott-silver) [reply .gitignore:13]: https://github.com/compound-finance/comet/blob/main/tasks/spider/task.ts#L5

Comment (kevincheng96) [inline .gitignore:13]: So contracts for all chains will live in the same directory? What happens if two contracts share the same address on different chains?

Comment (jflatow) [reply .gitignore:13]: I mean they can't, the addresses are deterministic

Comment (jflatow) [reply .gitignore:13]: If they share an address they are the same contract

Comment (kevincheng96) [reply .gitignore:13]: true

Comment (jflatow) [reply .gitignore:13]: Will follow on with a remove proxies PR hat touches this file, this is getting to large

**Pull request merged**